base/stream.jl

```diff
@@ -348,8 +348,29 @@ if Sys.iswindows()
     ispty(s::IO) = false
 end
 
-"    displaysize(io) -> (lines, columns)
-Return the nominal size of the screen that may be used for rendering output to this io object"
+"""
+    displaysize(io) -> (lines, columns)
+
+Return the nominal size of the screen that may be used for rendering output to
+this io object.
```

Comment:
I realize this is the existing wording unchanged, but I think `io` should be in uppercase, surrounded by backticks (e.g. `` `IO` ``) to make it clear that the input `io` has type `IO`.

Reply:
👌